spec/policy_machine_storage_adapters/active_record_spec.rb

```diff
@@ -181,9 +181,9 @@
               caffeinated.unassign(decaffeinated)
             end
 
-            expect(user.connected?(decaffeinated)).to be true
-            expect(user.connected?(caffeinated)).to be true
-            expect(caffeinated.connected?(decaffeinated)).to be false
+            expect(user.connected?(decaffeinated)).to be_truthy
```

Comment:
`be true` and `be_truthy` aren't the same. I think it's only suggested that methods ending in `?` return truthy or falsey values (as opposed to strictly `true` or `false`), but given that we're already testing for that level of detail, why not make these `eq true`?